azure-functions/acbs-function/api.js

```diff
@@ -217,6 +217,7 @@ const updateFacility = (facilityIdentifier, updateType, payload, etag) => put(`f
 
 // PATCH
 const updateFacilityLoan = (facilityIdentifier, loanId, payload) => patch(`facilities/${facilityIdentifier}/loans/${loanId}`, payload);
+const updateFacilityCovenant = (facilityIdentifier, payload) => patch(`facilities/${facilityIdentifier}/covenants`, payload);
```

Comment:
Can we update `api.api-test.js`?

Reply:
ive commonised and updated these -- i cant run dtfs locally atm so hopefully this works